scripts/test_dof.py

```diff
@@ -2,7 +2,7 @@
 
 from compas_ags.diagrams import FormGraph
 from compas_ags.diagrams import FormDiagram
-# from compas_ags.ags import graphstatics
+from compas_ags.ags import graphstatics
```

Comment:
why are you adding this if it is being loaded through the proxy?

Reply:
True...